web_services/foxx/brightid/index.js

```diff
@@ -235,8 +235,8 @@ const handlers = {
       }
     }
 
-    if (!db.isVerifiedFor(user, app, verification)) {
-      throw new errors.NotVerifiedError(contextId, appKey);
+    if (!db.isVerifiedFor(user, verification )) {
```

Comment:
`app.verification` should be passed if `verification` is an empty string or null.